src/python/platforms/android/adb.py

```diff
@@ -147,9 +167,32 @@ def run():
   return bytes_output.strip().decode('utf-8', errors='ignore')
 
 
+def copy_file(filename, src_dir, dest_dir, timeout=None):
+  """Copy file between two locations"""
+  file_src = os.path.join(src_dir, filename)
+  if environment.is_android_cuttlefish():
+    cvd_address = get_gce_address()
+    if not cvd_address:
```

Comment:
rather than duplicate these checks, can we just log_fatal inside `get_gce_address` ?